tests/CrEOF/Spatial/Tests/OrmTest.php

```diff
@@ -155,11 +160,15 @@ public static function setUpBeforeClass()
      */
     protected function setUp()
     {
+        if (count($this->supportedPlatforms) && ! in_array($this->getPlatform()->getName(), $this->supportedPlatforms, true)) {
+            $this->markTestSkipped(sprintf('No support for platform %s in test class %s.', $this->getPlatform()->getName(), get_class($this)));
+        }
+
         $this->entityManager = $this->getEntityManager();
         $this->schemaTool    = $this->getSchemaTool();
 
         if (true) {
-            static::getConnection()->executeQuery('SELECT 1 /*' . get_class($this) . '*/');
+            static::getConnection()->executeQuery(sprintf('SELECT 1 /*%s*//*%s*/', get_class($this), $this->getName()));
```

Comment:
Are you testing that the connexion is alive ? You should probably add a comment, and/or remove the `if(true)`